Sources/MessagingInApp/Provider/InAppProvider.swift

```diff
@@ -0,0 +1,26 @@
+import Common
+import Foundation
+import Gist
+
+// wrapper around Gist SDK to make it mockable
+internal protocol InAppProvider: AutoMockable {
+    func initialize(id: String, delegate: GistDelegate)
```

Comment:
in the other file initialize param was named `organizationId` probably want to be consistent